apps/www/pages/open-source/index.tsx

```diff
@@ -56,103 +93,119 @@ const OpenSource = () => {
+                          <Link key={i} href={`https://github.com/${x.sponsor}`}>
+                            <div className="rounded-2xl bg-scale-300 border-[1px] border-scale-400 p-6 cursor-pointer flex items-center space-x-3">
+                              <Image
+                                className="mt-4 mb-4 rounded-full w-12 h-12"
+                                src={`https://github.com/${x.sponsor}.png`}
+                                alt={`${x.sponsor} avatar`}
+                                width={45}
+                                height={45}
+                              />
+                              <h4>{x.sponsor}</h4>
+                            </div>
+                          </Link>
+                        ))}
+                      </div>
+                      <div className="w-full h-[1px] bg-scale-400 rounded-full mt-6"></div>
+                    </div>
+                  )
+              )}
+            </div>
+          </div>
+        </SectionContainer>
 
-              <div className="overflow-auto md:max-w-none hidden sm:block">
-                <ul className="flex 2xl:gap-4 items-center p-0">
-                  {maintainerPills.map((x) => (
-                    <li
-                      key={x}
-                      className={`mx-4 rounded-t-lg inline-block p-2 cursor-pointer hover:text-brand-800 ${
-                        activePill == x ? 'bg-gray-200 dark:bg-gray-400 text-brand-800' : ''
-                      }`}
-                      onClick={() => setActivePill(x)}
-                    >
-                      {x}
-                    </li>
-                  ))}
-                </ul>
-              </div>
-              <div className="grid lg:grid-cols-2 gap-4">
-                {maintainers
-                  .filter((x) => activePill == 'All' || x.tags.includes(activePill))
-                  .sort((a, b) => a.handle.localeCompare(b.handle))
-                  .map((x, idx) => (
-                    <div className="" key={idx}>
-                      <a
-                        className="flex gap-4"
-                        href={`https://github.com/${x.handle}`}
-                        target="_blank"
-                        rel="noreferrer"
-                      >
-                        <div className="grow bg-gray-300 dark:bg-gray-400 p-4 rounded-lg">
-                          <div className="flex gap-4 shrink-0">
+        <SectionContainer>
+          <div className="text-center space-y-2">
+            <h2 className="text-2xl">Community Maintainers</h2>
+            <p className="w-[50%] mx-auto text-scale-1100 text-sm pb-8">
+              Thanks to our dedicated community maintainers, we're able to propel our open source
+              mission forward. Their unwavering commitment drives our development, nurtures
+              collaboration, and empowers us to forge a more inclusive and accessible future.
```

Comment:
Kind of winged it for this so we might want to change it, unless its fine the way it is.

Reply:
Sounds good